README.md

```diff
@@ -58,12 +58,16 @@ contract('ERC20', function ([sender, receiver]) {
 
 ### Configuration
 
-This library features support for both web3 and truffle contract instances. The default environment is `'web3'`, unless a `'truffle'` environment is automatically detected. In a `'truffle`' environment, the web3 provider will be pulled from truffle's global web3 instance, otherwise, it defaults to `http://localhost:8545`.
+This library features support for both web3 and truffle contract instances: functions like `expectEvent` can receive receipts from either type of contract abstraction.
 
-While automatic detection should cover most use cases, both the environment and provider can be manually supplied:
+In a `truffle` environment, the web3 provider will be pulled from truffle's global web3 instance, otherwise, it defaults to `http://localhost:8545`.
```

Comment:
`truffle` was formatted as code here because it was a reference to the `environment` option value.

```suggestion
In a truffle environment, the web3 provider will be pulled from truffle's global web3 instance. Otherwise, it defaults to `http://localhost:8545`.
```

Reply:
This is a bit of a nitpick but it's part of a bigger issue, which is that after "removing" the environment option I think the readme was left in a frankestein state where things around environment and contract abstraction are not very clear.

Reply:
Yes, it did feel weird while editing it. Tbh I think we should rethink that whole section.

Reply:
Yeah I agree. If you have time do you want to give it a shot?